### PR TITLE
Adding scikit-mobility

### PR DESCRIPTION
**Pull request by gbarlacchi**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/scikit-mobility/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Use the SPDX identifier, e.g: GPL-2.0-only instead of GNU General Public License version 2.0
+  # See https://spdx.org/licenses/
+  license: BSD-3-Clause
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  license_family: BSD
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file
+  license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/scikit-mobility/meta.yaml:101]: I opened an issue upstream about manually packaging the license file:

https://github.com/scikit-mobility/scikit-mobility/issues/137

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/scikit-mobility/meta.yaml (hunk 1 above)
Comment (gbarlacchi) [reply recipes/scikit-mobility/meta.yaml:101]: Thanks! We'll add this in the new release.

File: recipes/scikit-mobility/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,122 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
[the remark this answers is not in the document]
Comment (gbarlacchi) [reply recipes/scikit-mobility/meta.yaml:1]: Thanks for your comments, we'll do the requested changes.

File: recipes/scikit-mobility/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    - skmob
+    - skmob.core
+    - skmob.io
+    - skmob.measures
+    - skmob.models
+    - skmob.preprocessing
+    - skmob.privacy
+    - skmob.tessellation
+    - skmob.utils
+  requires:
```
[the remark this answers is not in the document]
Comment (gbarlacchi) [reply recipes/scikit-mobility/meta.yaml]: Got it. Thanks, we've removed it.

File: recipes/scikit-mobility/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
```
[the remark this answers is not in the document]
Comment (gbarlacchi) [reply recipes/scikit-mobility/meta.yaml]: Agree, thanks. I've added it.

File: recipes/scikit-mobility/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
[the remark this answers is not in the document]
Comment (gbarlacchi) [reply recipes/scikit-mobility/meta.yaml]: Yes, you are right, thanks. I removed it.

File: recipes/scikit-mobility/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - folium
```
[the remark this answers is not in the document]
Comment (gbarlacchi) [reply recipes/scikit-mobility/meta.yaml]: Right, thank you. I made the change.

**Comment by synapticarbors** [comment]
Everything looks good. Just two final notes:
- In order for me to merge, I need confirmation that the other co-maintainers agree to be listed. They can either add a comment on this PR, or if you have communicated with them offline, you can let me know.
- Once the license file is in the source distribution, upon next release, please make sure you remove the manually added copy in the feedstock. I can help if that if you have questions.

**Comment by pellungrobe** [comment]
Yes, I confirm that I'll be a maintainer for this feed.

**Comment by jonpappalord** [comment]
Yes, I confirm that I'll be a maintainer of this feed.

**Comment by FilippoSimini** [comment]
I confirm that I'll be a maintainer for this feed.

**Comment by synapticarbors** [comment]
Thank you for your contribution to conda-forge. Let me know if you run into any issues once the package has a feedstock or feel free to reach out to the conda-forge team with questions.

**Pull request merged**